src/js/table.js

```diff
@@ -393,14 +393,28 @@ function reallyDisplayOrders(orders, beautiful) {
     return Promise.all(cell_value_promises).then( () => table );
 }
 
-function addCsvButton(orders) {
-    const title = 'download csv';
+function addCsvButtonP(orders) {
```

Comment:
Option A (preferred): add a bool param named output_format to addCsvButton
Option B (ok, but a bit smelly because of duplication): s/addCsvButtonP/addCsvButtonPlain/ and similarly s/addCsvButtonS/addCsvButtonSpreadsheet/.